### PR TITLE
Added a recommendation for Load balancer health probe

### PR DESCRIPTION
**Pull request by sonmitt**
# Overview/Summary

Created a new recommendation for Load balancer health probe with ARG query.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- Linked with feature AB[[31912](https://dev.azure.com/CSUSolEng/Well-Architected%20Framework/_workitems/edit/31912)]

## This PR fixes/adds/changes/removes

This PR will add a new recommendation in APRL v2 for Azure resources --> network --> loadBalancers

### Breaking Changes

None

## As part of this pull request I have

Output Screenshot below:

ARG query result for lb hp

- [ Y] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [ Y] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [Y ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [Y] Ensured PR tests are passing
- [Y] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [Y ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql
Comment (jimays-avila) [inline azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql:1]: @sonmitt I validate this query and it works. Maybe we need to add a not in the recommendation that this should be for LB's already deployed as new ones require creating a custom health probe.

**Review by jimays-avila — APPROVED** [review]

File: azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql
Comment (sonmitt) [reply azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql:1]: @jimays-avila 
Thanks Jimmy for reviewing. I am not sure where to add this note, can I add it in long description of the recommendation?

**Review by ejhenry — COMMENTED** [review]
Thanks for the contribution @sonmitt! Please include only a single learn more link for the recommendation.

File: azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql
Comment (jimays-avila) [reply azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql:1]: @ejhenry and @oZakari would be better to answer this.  Eric when you stated add a single learn more link is it referencing this?

Comment (sonmitt) [reply azure-resources/Network/loadBalancers/kql/e5f5fcea-f925-4578-8599-9a391e888a60.kql:1]: As per @ejhenry comment, I removed the "learn more" link from the recommendations.yaml.

**Review by ejhenry — APPROVED** [review]

**Pull request merged**